src/api/worker/offline/migrations/sys-v84.ts

```diff
@@ -0,0 +1,12 @@
+import { OfflineMigration } from "../OfflineStorageMigrator.js"
+import { OfflineStorage } from "../OfflineStorage.js"
+import { migrateAllElements } from "../StandardMigrations.js"
+import { createMissedNotification, MissedNotificationTypeRef } from "../../../entities/sys/TypeRefs.js"
+
+export const sys84: OfflineMigration = {
+	app: "sys",
+	version: 84,
+	async migrate(storage: OfflineStorage) {
+		await migrateAllElements(MissedNotificationTypeRef, storage, [createMissedNotification])
```

Comment:
missed notifications are not cached (they don't even really exist haha)

Reply:
we put a no-op migration with an explanation because the compile time check does not know about this